README.md

```diff
@@ -7,8 +7,48 @@ openSSL module as extension for chassis.io. See https://github.com/Chassis/Chass
 1. Run `vagrant reload --provision` or just `vagrant up`
 1. Profit.
 
-## Caveats and known problems:
+## Notes:
 
-1. I haven't tested with multisite.
-1. You also need to modify the `wp-config.php` in the root directory and modify the `WP_SITEURL` and `WP_HOME` constants to use https instead of http
-1. You'll have to add the `.cert` file in the root directory of Chassis to your keychain and set it to "Always trust" (right click on it, Get Info) to avoid the red "your connection is not encrypted" message. Don't know how to do it on a Windows machine. Let me know :)
+1. You also need to modify the `wp-config.php` in the root directory and modify the `WP_SITEURL` and `WP_HOME` constants to use https instead of http.
+1. Both the certificate and key are exported to `chassis/` directory, so it can be used with local dev servers, like webpack dev server.
+
+## Trusting the certificate
+
+In order to avoid security errors and get that nice green padlock in your location bar, you should add the site's SSL certificate to your trust store. The certificate can be found at chassis/{DOMAIN_HERE}.cert .
+
+### macOS via CLI
+`sudo security add-trusted-cert -d -r trustRoot -k /Library/Keychains/System.keychain chassis/{DOMAIN_HERE}`
+
+### Firefox on any operating system:
+
+Open Firefox's Preferences.
+Go to Advanced -> Certificates -> View certificates -> Authorities.
+Import the certificate.
+Click "Trust this CA to identify web sites".
+
+### Chrome or Safari on Mac:
+
+- Open the "Keychain Access" app.
+- Drag the certificate into the "System" keychain.
+- Right-click it and click "Get Info".
+- Expand the "Trust" section if it's not already.
+- Under "Secure Sockets Layer (SSL)", select "Always Trust".
+- Close the window. At this point you may have to enter your Mac account password.
+- Restart your browser for this to take effect.
+
+### IE and Edge on Windows ( [reference](https://medium.com/@ali.dev/how-to-trust-any-self-signed-ssl-certificate-in-ie11-and-edge-fa7b416cac68) )
+
+> - Browse to your page (e.g. https://vagrant.local) in Internet Explorer which should use your self-signed SSL certificate. You should be greeted by an error message saying your certificate is not trustworthy.
+> - Click “Continue to this website”.
+> - Click on “Certificate error” in the address bar, and then click “View certificates”.
+> - Click “Install Certificate”.
+> - Click “Place all certificates in the following store”, and then click “Browse”. Do not rely on the preselected option to automatically select the certificate store as this will not work!
+> - Inside the dialog box, click “Trusted Root Certification Authorities”, and then click “OK”.
+> - Finish the dialog.
+> - When you get a security warning, click “Yes” to trust the certificate.
+> - Reload your page. The certificate should be working fine now.
+
```

Comment:
Could add Windows command line option too:

https://github.com/humanmade/altis-local-chassis/blob/master/inc/class-command.php#L234

When running the command prompt or some other terminal app as administrator you can do:

```
certutil -enterprise -f -v -AddStore "Root" "vagrant.local.cert"
```